README.md

```diff
@@ -29,81 +29,10 @@ In most cases, running `./install.sh` should be sufficient to install all packag
 - `--py-ver=VERSION`: Replace the Python version specified in the conda environment file with `VERSION` (e.g., `--py-ver=3.10`).
 - `--help`: Print this help text.
 
-## Testing the installation
+# Testing the installation
 
 Testing can be done by:
 
 ```bash
 $ mirgecom/examples/run_examples.sh mirgecom/examples/
 ```
-
-## Running on systems with lots of nodes (>256)
-On large systems, the file system can become a bottleneck for loading Python
-packages. On these systems, it is recommended to create a zip file with the
-modules to speed up the startup process. This can be done by specifying the
-`--modules` parameter to `install.sh`, or by running `makezip.sh` after
-installation.
-
-See https://github.com/illinois-ceesd/planning/issues/26 for more details.
```

Comment:
Is this potentially something we still need to worry about? If so, should we make an issue? (Or we could just wait until someone runs into it again I guess...)

Reply:
We haven't really encountered this "in the wild" I think. Startup time is mostly determined by compile time currently.